pkg/network/tracer/offsetguess/tracer.go

```diff
@@ -576,7 +577,12 @@ func (t *tracerOffsetGuesser) Guess(cfg *config.Config) ([]manager.ConstantEdito
 		What:         uint64(GuessSAddr),
 	}
 
-	if !cfg.CollectIPv6Conns {
+	kv, err := kernel.HostVersion()
+	if err != nil {
+		return nil, err
+	}
+	guessIPv6 := cfg.CollectIPv6Conns && kv < kernel.VersionCode(5, 18, 0)
```

Comment:
I thought this was only for flowi6, i.e. `ip6_make_skb`?

Reply:
Trying to make ci pass for this one I guess?

Reply:
Let me double check if any tests fail without having `flowi6` offsets.